src/lib/parsers/sol-json.c

```diff
@@ -512,23 +512,15 @@ sol_json_escape_string(const char *str, char *buf, size_t len)
     return out;
 }
 
-SOL_API int
-sol_json_double_to_str(const double value, char *buf, size_t len)
+static void
+double_to_str(char *buf, size_t len)
```

Comment:
as for the size, I recall it was 66 bytes for the number + two for exponent and sign encoding plus 3 for the actual exponent value, then one for the trailing null byte. That's: 66+2+3 = 71. Make it 100 and should be enough.